Gemfile.lock

```diff
@@ -398,23 +398,23 @@ GEM
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
-    sshkit (1.20.0)
+    sshkit (1.21.0)
```

Comment:
### [sshkit](http://github.com/capistrano/sshkit)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.20.0 → 1.21.0

[_[change-log](https://github.com/capistrano/sshkit/releases), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/capistrano/sshkit/compare/v1.20.0...v1.21.0).

These are the individual commits:
- (b245b18) [Automatically maintain release notes w/ GH Actions](https://github.com/capistrano/sshkit/commit/b245b181984aec9ac71c1868a8844f4eec109e09)
- (986331e) [Merge pull request #472 from capistrano/chores/use-github-ac…](https://github.com/capistrano/sshkit/commit/986331ee352a385538d2d1f0f6436bc0ed7237e4)
- (d5a3e63) [Add changelog_uri to metadata to easily link from rubygems.o…](https://github.com/capistrano/sshkit/commit/d5a3e6306a90ab87eeade40b86dfc37b8b0cee01)
- (af9cefd) [Removed dead links from README (#477)](https://github.com/capistrano/sshkit/commit/af9cefd53644dd5f9c3c11f66b47e6afc1fad8d4)
- (9166ea4) [Pin mocha at 1.9.x to fix CI failures (#478)](https://github.com/capistrano/sshkit/commit/9166ea4b2bedd158ada7e3f986f918656ce6d82a)
- (d69522c) [Unpin mocha version and fix deprecation warnings for mocha v…](https://github.com/capistrano/sshkit/commit/d69522c98c963c314bdda8cf404e337ceb7b9394)
- (810651d) [Command: wrap a Proc interaction_handler into a MappingInter…](https://github.com/capistrano/sshkit/commit/810651d84b65991971404e040ee8ccfd532897e4)
- (a4ad4b6) [Release 1.21.1](https://github.com/capistrano/sshkit/commit/a4ad4b6dc9ee62452988c28a6dc5618d258871fa)


</details>